src/gql/utils/defaultObjects.ts

```diff
@@ -1,6 +1,5 @@
 import {
```

Comment:
Most of these deleted objects can be used for the non-specified cases unless the object type was changed

Reply:
https://github.com/NibiruChain/ts-sdk/pull/347#discussion_r1580077123

Reply:
https://github.com/NibiruChain/ts-sdk/pull/347#discussion_r1581706548